articles/tokens/concepts/refresh-token-rotation.md

```diff
@@ -0,0 +1,45 @@
+---
+description: Understand how Refresh Token rotation provides greater security by issuing a new Refresh Token with each request made to Auth0 for a new Access Token by a client using Refresh Tokens.
+topics:
+  - tokens
+  - refresh-token-rotation
+contentType: concept
+useCase:
+  - refresh-token-rotation
+  - silent-authentication
+---
+# Refresh Token Rotation
+
+[Refresh Token](/tokens/concepts/refresh-tokens) rotation provides greater security by issuing a new Refresh Token and invalidating its predecessor token with each request made to Auth0 for a new [Access Token](/tokens/cocncepts/access-tokens). 
+
+Refresh Token rotation in Auth0 conforms with the [OAuth 2.0 BCP](https://tools.ietf.org/html/draft-ietf-oauth-security-topics-13#section-4.12). 
+
+Refresh Token rotation is supported in conjunction with the following flows:
+* [Authorization Code Flow](/flows/concepts/auth-code)
+* [Authorization Code Flow with Proof Key for Code Exchange (PKCE)](/flows/concepts/auth-code-pkce)
+* [Resource Owner Password Credentials Exchange](/api-auth/tutorials/adoption/password)
+* [Device Authorization Flow](/flows/concepts/device-auth)
+
+## How it works
+
+Typically, ID and Access Tokens are obtained from the authorization server and cached in memory. Token renewal (due to evicted memory or expiration) is handled by the SDK, relieving the developer of this burden. Silent re-authentication is technically achieved by sending a `prompt=none` parameter on the authentication request and using a hidden iframe, provided that there is still an active user session on the authorization server. This method, however, is impacted by [Intelligent Tracking Prevention (ITP)](https://webkit.org/blog/7675/intelligent-tracking-prevention/). 
+
+Starting with ITP2, this became a problem with sessions being prematurely terminated in Safari. This is especially a concern if you want to offer long-lived sessions. To address this issue, you can use Refresh Token rotation to leverage the use of Refresh Tokens in the browser that adhere to the OAuth 2.0 BCP. 
```

Comment:
This is also becoming an issue with other browsers too, such as Chrome.

Reply:
The other one that people have reported to us through GitHub is Brave with it's Shield technology up (blocks third-party cookies by default).

Reply:
> due to evicted memory or expiration

It might not be obvious what 'evicted memory' really means here, might be worth adding "refreshing the browser" to this list.

Reply:
Clarified to "memory cache eviction budgets" and added "refreshing the browsers"